language/cs.json

```diff
@@ -188,6 +188,12 @@
               "label": "Show as hotspot?",
               "description": "Show as a hidden hotspot, can used for overlaying interaction on elements in image"
             },
+            {
+              "label": "Show hotspot when mouse is hovering?"
```

Comment:
Are the question marks necessary? Do the settings usually have them?

Reply:
Good point, the should probs be more imperative than interrogative

Reply:
I definitely know what those words mean 🧠